src/components/ProgressIndicator/ProgressIndicatorDisplay.js

```diff
@@ -108,17 +110,37 @@ const ProgressIndicatorDisplay = ({
 }) => {
   const [expanded, setExpanded] = useState(false);
 
+  const intl = useIntl();
+  // Wrapper may be a DOM element, which can't handle <FormattedMessage />
+  const ariaIconLabel = intl.formatMessage({
+    description: 'Toggle active step progress indicator icon',
+    defaultMessage: 'Progress step indicator icon',
+  });
+  const toggleStepsTitle = intl.formatMessage({
+    description: 'Active step toggle titles',
+    defaultMessage: 'Show/hide steps',
+  });
+
   return (
     <Card blockClassName="progress-indicator" modifiers={expanded ? [] : ['mobile-collapsed']}>
-      <div
+      {/* Turn a Div with an onClick event into a button for accesibility */}
+      <button
         className={getBEMClassName('progress-indicator__mobile-header')}
+        aria-pressed={expanded ? 'true' : 'false'}
         onClick={() => setExpanded(!expanded)}
       >
-        <FAIcon icon={expanded ? 'chevron-up' : 'chevron-down'} modifiers={['normal']} />
-        <span className={getBEMClassName('progress-indicator__active-step')}>
+        <FAIcon
+          icon={expanded ? 'chevron-up' : 'chevron-down'}
+          modifiers={['normal']}
+          aria-label={ariaIconLabel}
+        />
+        <span
+          className={getBEMClassName('progress-indicator__active-step')}
+          title={toggleStepsTitle}
```

Comment:
shouldn't this be `aria-label` instead of `title` so that screen readers pick this up?